src/main/scala/kiteytys/conf/Conf.scala

```diff
@@ -18,6 +18,7 @@ object Conf {
     val user     = conf.getString("username")
     val password = conf.getString("password")
     val ssl      = conf.getBoolean("ssl")
+    val admin    = conf.getString("admin")
```

Comment:
Small thing but could this be `adminEmail` instead of `admin`. Maybe easier to understand what this is for.